### DISCUSSION TITLE
Error installing version 24.2.2

### DISCUSSION THREAD

**Discussion by uniplan-sw** [dbeaver/dbeaver#35841] [Q&A]
Up to now I have updated the dbEaver versions without any problems. Today, when I update to version 24.2.2, when I launch the software I get an error message. This is the log:

`!SESSION 2024-10-07 08:47:17.352 -----------------------------------------------
eclipse.buildId=unknown
java.version=17.0.6
java.vendor=Eclipse Adoptium
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=it
Framework arguments:  -eclipse.keyring C:\Users\vs2010\AppData\Roaming\DBeaverData\secure\secure_storage
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.osgi 4 0 2024-10-07 08:47:41.834
!MESSAGE An error occurred while automatically activating bundle org.eclipse.ui.ide (164).
!STACK 0
org.osgi.framework.BundleException: Exception in org.eclipse.ui.internal.ide.IDEWorkbenchPlugin.start() of bundle org.eclipse.ui.ide.
	at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:854)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:775)
`

**Comment by E1izabeth** [comment]
What Windows version do you use?

**Reply by uniplan-sw** [reply]
windows 2016 server standard

**Reply by TB-HunBoka** [reply]
Can it relate to this: https://github.com/eclipse-platform/eclipse.platform.swt/issues/1500 ?
Is there any minor windows update that you can install?

**Reply by uniplan-sw** [reply]
I looked at the document but I couldn't find any clues for my problem. Could you please explain better what to do? Thanks.

**Reply by E1izabeth** [reply]
The platform on which DBeaver is based stopped support for Windows Server 2016 https://github.com/dbeaver/dbeaver/issues/35694
There are two ways to solve the issue:
- Downgrade DBeaver to 24.2.0
- Try upgrading Windows. If no additional minor upgrades are available, upgrade to a higher major release of Windows.

**Comment by uniplan-sw** [comment]
Now it's clear. Thank you.
